clipmenu

```diff
@@ -1,7 +1,17 @@
 #!/bin/bash
 
-: "${CM_LAUNCHER=dmenu}"
 : "${CM_DIR="${XDG_RUNTIME_DIR-"${TMPDIR-/tmp}"}"}"
+# Pick dmenu or rofi, depending on what's available on the system
+if [[ -z "$CM_LAUNCHER" ]]; then
+  if command -v dmenu >/dev/null 2>&1; then
+    CM_LAUNCHER=dmenu
```

Comment:
indentation in this patch is wrong, it should be 4 spaces per indent.

Reply:
Yeah, my bad, I wrote that in a hurry... I'll fix it tomorrow!